repo/incubating/flink/versions/0/flink-frameworkversion.yaml

```diff
@@ -53,9 +115,21 @@ spec:
           spec:
             containers:
             - name: taskmanager
-              image: flink:1.7
+              image: kudobuilder/flink:1.7.2
```

Comment:
https://github.com/docker-flink/docker-flink/pull/68

I built an image with this PR (and the changes I suggested in the comment).